packages/frontend/src/components/Files/File.tsx

```diff
@@ -32,6 +32,7 @@ export const NodeItem = ({ node, row }: any) => {
   const { id, name, modifiedAt, content } = entry;
   const contentMimeType = get(content, 'mimeType', '');
   const contentSizeInBytes = get(content, 'sizeInBytes', 0);
+  const { REACT_APP_ALFRESCO_DOWNLOAD_URL } = process.env;
```

Comment:
Haven't seen process.env used like this. Intriguing! What is the value here if that variable is not in env?

Reply:
it's `undefined`